plugins/lighthouse/src/components/CreateAudit/index.tsx

```diff
@@ -113,7 +113,7 @@ const CreateAudit: FC<{}> = () => {
           <Grid item xs={12} sm={6}>
             <InfoCard>
               <form
-                onSubmit={(ev) => {
+                onSubmit={ev => {
```

Comment:
Not sure if this is desired? Seems like prettier is configured badly in some places if so

Reply:
This is how we set it afaik

Reply:
@freben sorry let me rephrase, should we not lint at build time still?

I mean I like the fact that it doesn't complain for stuff that you haven't changed, but it just makes me worried only the most maintained files will have the latest lint standards. And when you have a lint rule that prefers something like `import {...} from '@backstage/core'` over `../../../packages/core` it can make simple renaming tasks a pain.